IfcDistributionSystem Rail Relocated Pset_EnvironmentalCondition (#312)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/s/SaltMist_3rzK_24B55FhlIh99zlW26/Documentation.md

> Maximum level of salt mist that the equipment complies with

Reviewer: ```suggestion
Maximum level of salt mist that the equipment complies with.
```

> Maximum level of salt mist that the equipment complies with

Reviewer 2: ```suggestion
Maximum level of salt mist that the equipment can operate effectively in.
```
